spec/dummy/app/controllers/application_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationController < ActionController::Base
```

Comment:
https://github.com/rails/rails/tree/master/guides/bug_report_templates

I'm not sure there's one that would be appropriate here or not.